src/ert/gui/tools/file/file_dialog.py

```diff
@@ -77,21 +78,23 @@ def _quit_thread(self) -> None:
         self._file.close()
 
     def _init_layout(self) -> None:
-        dialog_buttons = QDialogButtonBox(QDialogButtonBox.StandardButton.Ok)
-        dialog_buttons.accepted.connect(self.accept)
+        dialog_buttons = QDialogButtonBox(QDialogButtonBox.StandardButton.Close)
+        dialog_buttons.rejected.connect(self.reject)
 
         self._copy_all_button = dialog_buttons.addButton(
             "Copy all",
             QDialogButtonBox.ButtonRole.ActionRole,
         )
-        self._copy_all_button.clicked.connect(self._copy_all)  # type: ignore
+        assert self._copy_all_button is not None
```

Comment:
is this assert necessary?

Reply:
Yup, to use @eivindjahren's words - make implicit assumptions explicit..